com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -66,10 +66,17 @@ public static ActionSpec MakeDiscrete(params int[] branchSizes)
             return actuatorSpace;
         }
 
-        internal ActionSpec(int numContinuousActions, int[] branchSizes = null)
+        /// <summary>
+        /// Create an ActionSpec initialized with the specified action sizes.
+        /// </summary>
+        /// <param name="numContinuousActions">The number of continuous actions available.</param>
+        /// <param name="discreteBranchSizes">The array of branch sizes for the discrete actions.  Each index
+        /// contains the number of actions available for that branch.</param>
+        /// <returns>An ActionSpec initialized with the specified action sizes.</returns>
+        public ActionSpec(int numContinuousActions = 0, int[] discreteBranchSizes = null)
```

Comment:
MakeDiscrete uses the `params int[]` syntactic sugar, but I feel like adding that here is potentially confusing.